pyproject.toml

```diff
@@ -12,6 +12,7 @@ dependencies = [
     "pydantic-settings>=2.8.1",
     "platformdirs>=4.3.7",
     "PyYAML>=6.0.2",
+    "anyio>=4.10.0",
```

Comment:
doesn't look like we are using this yet. maybe hold off adding it until we do, but fine leaving it in bc we will eventually anyway